pkg/security/secl/model/utils.go

```diff
@@ -74,13 +80,21 @@ func UnmarshalPrintableString(data []byte, size int) (string, error) {
 		return "", ErrNotEnoughData
 	}
 
-	str, err := UnmarshalString(data, size)
-	if err != nil {
-		return "", err
+	if len(data) < size {
+		return "", ErrNotEnoughData
+	}
+
+	i := bytes.IndexFunc(data[:size], func(r rune) bool {
```

Comment:
Thanks for the optimization. Do you think that we could move the implementation in the IsPrintable function as it is used in other places.

Reply:
After this change, `IsPrintable` is not used anywhere else. I can remove it if that's ok with you.